### PR TITLE
MS-4, MS-6, MS-10: Group searchable variant effects into translatable categories

### PR DESCRIPTION
**Pull request by sh-ura**
Authored with @marta- and @sdumitriu

### CODE REVIEW

**Comment by sh-ura** [comment]
@sdumitriu I didn't notice your comment earlier and have since rebased the commits so the context for it was lost.... Could you please clarify which line you're referring to? 
The comment was something life "If it doesn't have an `@Role` then it's useless."

File: api/src/main/java/org/phenotips/mendelianSearch/internal/MendelianVariantCategory.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ *
+ * A MendelianVariantCategory object describes a single category of variants that
+ * may be used to filter searches in the Mendelian Search Application.
+ * A variant category contains multiple variant effects that are used to query the
+ * variant store.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Component
```
Comment (sdumitriu) [inline api/src/main/java/org/phenotips/mendelianSearch/internal/MendelianVariantCategory.java]: This annotation only makes sense if it's supposed to be an injectable component implementation, in which case it must also have some other class or interface annotated with `@Role` in its hierarchy. Just remove this annotation.

**Review by sdumitriu — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/phenotips/mendelianSearch/internal/MendelianVariantCategory.java (hunk 1 above)
Comment (sh-ura) [reply api/src/main/java/org/phenotips/mendelianSearch/internal/MendelianVariantCategory.java]: Ok thank you!

**Comment by katpavlova** [comment]
@sdumitriu Annotation has been removed. @veronikaslc I found some bugs so I closed the PR but they have since been fixed, I have tested the PR quite thoroughly so please review the code when you get a chance to.

**Pull request closed without merging**